web/css/components/modal.css

```diff
@@ -1,14 +1,27 @@
-.modal--content i {
-  margin-right: 20px;
+body {
```

Comment:
delete style applied to body, if you need to set somes, set the style in the html for demo purpose